core/src/main/java/org/elasticsearch/index/analysis/SynonymGraphTokenFilterFactory.java

```diff
@@ -79,7 +79,7 @@ public String name() {
         @Override
         public TokenStream create(TokenStream tokenStream) {
             // fst is null means no synonyms
-            return synonymMap.fst == null ? tokenStream : new SynonymGraphFilter(tokenStream, synonymMap, ignoreCase);
+            return synonymMap.fst == null ? tokenStream : new SynonymGraphFilter(tokenStream, synonymMap, false);
```

Comment:
Hmm, I see that the field `ignoreCase` is deprecated but this change looks unrelated to this PR? Looks like you removed this field together with this PR (I think a separate commit would have been better but it's fine with me).

Reply:
I'll split into a separate commit